Package.swift

```diff
@@ -0,0 +1,34 @@
+// swift-tools-version: 6.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "XRPLSwift-rest",
+    platforms: [
+        .iOS(.v13),
+        .macOS(.v10_15)
+    ],
+    products: [
+        .library(
+            name: "XRPLSwift-rest",
+            targets: ["XRPLSwift-rest"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/Flight-School/AnyCodable.git", from: "0.6.7")
```

Comment:
It causes trouble when the same package is used in the project but with different version. An option like `.upToNextMajor` might help.

Reply:
[7a28bb6](https://github.com/girin-app/XRPLSwift-rest/pull/1/commits/7a28bb6df02bc30119868be6a7e1be14e8d8cf19)